tests/framework/mail/BaseMailerTest.php

```diff
@@ -45,18 +43,19 @@ protected function getTestFilePath()
     }
 
     /**
-     * @return Mailer test email component instance.
+     * @return TestMailer test email component instance.
      */
     protected function createTestMailComponent()
     {
-        $component = new Mailer();
-        $component->viewPath = $this->getTestFilePath();
-
-        return $component;
+        return new TestMailer([
+            'composer' => [
+                'viewPath' => $this->getTestFilePath()
```

Comment:
Should we add comma after last array element if array elements are split to separate lines? Even in tests? [According core code style guide section](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#associative) is a bit unclear about it.